adserver/migrations/0063_data_stripe_fk_migration.py

```diff
@@ -0,0 +1,53 @@
+"""
+Migrates the existing model attributes:
+ - Advertiser.stripe_customer_id
+ - Publisher.stripe_connected_account_id
+
+This migration *MUST* be run after ./manage.py djstripe_sync_models
+"""
+from django.db import migrations
+
+
+def forwards(apps, schema_editor):
+    """Migrate to proper Stripe FKs."""
+    Advertiser = apps.get_model("adserver", "Advertiser")
+    Publisher = apps.get_model("adserver", "Publisher")
+    Customer = apps.get_model("djstripe", "Customer")
+    Account = apps.get_model("djstripe", "Account")
+
+    for advertiser in Advertiser.objects.all():
+        if advertiser.stripe_customer_id:
+            advertiser.djstripe_customer = Customer.objects.get(id=advertiser.stripe_customer_id)
```

Comment:
I wonder if we need to be defensive here at all and check for `None`? Perhaps `filter().first()`?

It seems like there could be old/outdated id's in the DB that aren't in Stripe for some reason.

Reply:
I purposefully made this *not* defensive. I want it to blow up loudly (fail the migration and force data correction) if something like this occurs.